File: Data Structures & Algorithms/two-integer-sum-ii/submission-1.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& numbers, int target) {
        int n=numbers.size();
        vector<int> ans;
        for(int i=0;i<n-1;i++)
        {
            for(int j=i+1;j<n;j++)
            {
                if(i!=j && numbers[i]+numbers[j]==target)
                {
                ans.push_back(i+1);
                ans.push_back(j+1);
                }
            }
        }
        return ans;
        
    }
};

File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        ans=[]
        mpp={}
        for i in range(len(nums)):
            book=target-nums[i]
            if book in mpp:
                ans.append(mpp[book])
                ans.append(i)
            else:
                mpp[nums[i]]=i

        return ans

        